### PR TITLE
[BACKPORT] Fix create existing operand actor

### PR DESCRIPTION
**Pull request by NJUPole**
This PR is a backport of #228 and part of #301 . 

Because of the refactor of operand actor and graph actor, the code and logic may be different from the original commit.

### CODE REVIEW

File: mars/scheduler/graph.py [hunk 1]
```diff
@@ -625,33 +653,11 @@ def create_operand_actors(self, _clean_io_meta=True):
...
-                shared_input_chunks=list(shared_input_chunk_keys),
-                chunks=list(chunks),
-            )
+            io_meta = self._collect_operand_io_meta(chunk_graph, self._op_key_to_chunk[op_key])
             op_info['op_name'] = op_name
             op_info['io_meta'] = io_meta
 
-            if predecessor_keys:
+            if io_meta['predecessors']:
                 state = 'UNSCHEDULED'
```
Comment (wjsi) [inline mars/scheduler/graph.py]: This can be changed into real enum value instead of names and values to reduce code complexity.

Comment (NJUPole) [reply mars/scheduler/graph.py]: Done.

File: mars/scheduler/operand.py [hunk 2]
```diff
@@ -118,6 +119,38 @@ def add_finished_predecessor(self, op_key):
...
+            self._pred_keys.update(op_info['io_meta']['predecessors'])
+            self._succ_keys.update(op_info['io_meta']['successors'])
+
+        futures = []
+        for pred_key in op_info['io_meta']['predecessors']:
+            futures.append(self._get_operand_actor(pred_key).remove_finished_succs(
+                self._op_key, _tell=True, _wait=False))
+        for succ_key in op_info['io_meta']['successors']:
+            futures.append(self._get_operand_actor(succ_key).remove_finished_preds(
+                self._op_key, _tell=True, _wait=False))
```
Comment (wjsi) [inline mars/scheduler/operand.py]: It is better to use a batch interface

File: mars/scheduler/operand.py [hunk 3]
```diff
@@ -118,6 +119,38 @@ def add_finished_predecessor(self, op_key):
...
+        [future.result() for future in futures]
+
+    def remove_finished_succs(self, op_key):
```
Comment (wjsi) [inline mars/scheduler/operand.py]: Comments needed to mark these as workarounds.

**Pull request merged**